content/statsServiceMap.md

```diff
@@ -7,7 +7,6 @@ title: Using the dataRetrieval Stats Service
 categories: Data Science
 tags: 
   - R
-  - dataRetrieval
```

Comment:
I know it's a pain, but we do need to manually enter anything past R...and I'm a big fan of keeping "dataRetrieval" as a tag here